src/src/tx_main.cpp

```diff
@@ -744,7 +744,6 @@ void setup()
   #ifdef ENABLE_TELEMETRY
   TelemetryReceiver.SetDataToReceive(sizeof(CRSFinBuffer), CRSFinBuffer, ELRS_TELEMETRY_BYTES_PER_CALL);
   #endif
-  POWERMGNT.setDefaultPower();
```

Comment:
Powermgmt init must be moved after radio init to make sure the SPI is initialized

Reply:
Not sure I understand this one? I haven't moved the init on Powermgmt ?

Reply:
You moved `setDefaultPower();` to end of the init() func. That calls radio setpower which requires SPI to be configued :)

Reply:
Ahhh gotcha :) Move it down a bit.